app/events.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+  If space or enter is pressed while focused on an element then click that
+  element.
+*/
+function clickOnSpaceOrEnter(event: KeyboardEvent) {
+  if (event.key === 'Space' || event.key === 'Enter' ||
+    event.keyCode === 32 || event.keyCode === 13
+  ) {
+    _clickElement(event.target as HTMLElement);
+  }
+}
+
+// Send a mouse click event to an element
+function _clickElement(element: HTMLElement) {
+  const event = document.createEvent('MouseEvent');
+  event.initMouseEvent('click', true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null);
```

Comment:
The arguments to this method are unreadable. 5 `0`s in a row followed by 4 `false`s without an explanation of what they mean is so uncool man

Reply:
Ill leave a comment here. I was using a much more obvious way to create a mouse event, but this is the only way to do it in IE, with unnamed countless parameters.